### PR TITLE
rerun cfret plates and extract single cells

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
@gwaybio 

Here are the rerun plates with extracted single cell features.

### CODE REVIEW

File: 2.cellprofiler_processing/CellProfiler_input/CFReT_project_CL.cppipe [hunk 1]
```diff
@@ -173,10 +173,10 @@ MeasureGranularity:[module_num:9|svn_version:'Unknown'|variable_revision_number:
 
 MeasureObjectIntensity:[module_num:10|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
     Select images to measure:Actin, ER, Golgi, Hoechst, Mitochondria
-    Select objects to measure:Cells, Cytoplasm, Nuclei, OrigNuclei
+    Select objects to measure:Cells, Cytoplasm, Nuclei
 
 MeasureImageIntensity:[module_num:11|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
-    Select images to measure:Actin, ER, Golgi, Hoechst
+    Select images to measure:Actin, ER, Golgi, Hoechst, Mitochondria
```
Comment (gwaybio) [inline 2.cellprofiler_processing/CellProfiler_input/CFReT_project_CL.cppipe:179]: nice! It seems you caught this bug too?

**Review by gwaybio — COMMENTED** [review]
One comment, but before I approve, can you add the `nbconverted` file for `extract_sc_features.ipynb`?

File: 2.cellprofiler_processing/CellProfiler_input/CFReT_project_CL.cppipe (hunk 1 above)
Comment (jenna-tomkinson) [reply 2.cellprofiler_processing/CellProfiler_input/CFReT_project_CL.cppipe:179]: Ahh darn my bad. And yes I did catch it! I do notice sometimes when using the CellProfiler GUI that is will randomly unselect things, which was something I discovered in the NF1 pipelines.

Let me convert it now!

**Comment by jenna-tomkinson** [comment]
@gwaybio The file is converted now.

**Review by gwaybio — APPROVED** [review]
👍 

Adding details (or pointing to them) about what plates are what in the sc features notebook would be helpful. No need for me to re-review

**Pull request merged**